src/components/organism/SwapTokens/index.tsx

```diff
@@ -87,7 +96,9 @@ const SwapTokens = () => {
   const [slippageAuto, setSlippageAuto] = useState<boolean>(true);
   const [slippageValue, setSlippageValue] = useState<number>(15);
   const [walletHasEnoughWnd, setWalletHasEnoughWnd] = useState<boolean>(false);
-  const [availablePoolTokens, setAvailablePoolTokens] = useState<any[]>([]);
+  const [availablePoolTokenA, setAvailablePoolTokenA] = useState<any[]>([]);
+  const [availablePoolTokenB, setAvailablePoolTokenB] = useState<any[]>([]);
```

Comment:
Do u have a type for this availablePoolTokenB and availablePoolTokenB? Instead of any[] can u use type?